app/controllers/admin/podcasts_controller.rb

```diff
@@ -5,7 +5,7 @@ def create
       authorize_resource(resource)
 
       if resource.save
-        Podcasts::GetEpisodesJob.perform_later(podcast_id: resource.id)
+        Podcasts::GetEpisodesJob.perform_later(podcast_id: resource.id) if resource.published
```

Comment:
should this be triggered everytime a Podcast is saved?

if I open the admin, don't touch anything, save the page, this will fetch the episodes, maybe we should only fetch them if the published attribute goes from false to true?

I'm not sure, maybe it's better to always fetch them

Reply:
Episodes are only fetched on create (this code is inside the `create` action).
When a podcast goes from unpublished to published, episodes will be fetched via rake task which is executed every hour.